server/types/callbacks.go

```diff
@@ -28,28 +28,57 @@ type Callbacks interface {
 	//   non-zero value to indicate the rejection reason (typically 401, 429 or 503).
 	//   HTTPResponseHeader may be optionally set (e.g. "Retry-After: 30").
 	//   ConnectionCallbacks is ignored.
-	OnConnecting(request *http.Request) ConnectionResponse
+	OnConnecting func(request *http.Request) ConnectionResponse
+}
+
+func (c *Callbacks) SetDefaults() {
+	if c.OnConnecting == nil {
+		c.OnConnecting = func(r *http.Request) ConnectionResponse {
+			return ConnectionResponse{Accept: true}
+		}
+	}
 }
 
 // ConnectionCallbacks receives callbacks for a specific connection. An implementation of
 // this interface MUST be set on the ConnectionResponse returned by the OnConnecting
 // callback if Accept=true. The implementation can be shared by all connections or can be
 // unique for each connection.
```

Comment:
Can we change the wording here because it still refers to "interface" and "implementation" here?